test/puppetlabs/puppetdb/pdb_routing_test.clj

```diff
@@ -52,7 +48,7 @@
 
       (let [resp (export svc-utils/*base-url*)]
         (tu/assert-success! resp)
-        (is (.contains (get-in resp [:headers "Content-Disposition"]) "puppetdb-export"))
+        (is (.contains (get-in resp [:headers "content-disposition"]) "puppetdb-export"))
```

Comment:
I think this might be where it's happening 
https://github.com/puppetlabs/clj-http-client/blob/master/src/clj/puppetlabs/http/client/async.clj#L192